packages/graphql/src/schema/resolvers/wrapper.ts

```diff
@@ -147,3 +151,15 @@ async function decodeToken(
     }
     return undefined;
 }
+
+function verifyGlobalAuthentication(
+    context: SubscriptionContext | Context,
+    plugin: Neo4jGraphQLAuthPlugin | undefined
+): void {
+    const isGlobalAuthenticationEnabled = plugin?.getGlobalAuthenticationEnabled();
+    if (isGlobalAuthenticationEnabled) {
+        if (!context.jwt) {
+            throw new Neo4jGraphQLAuthenticationError("Enabled global authentication requires a valid JWT token");
+        }
+    }
+}
```

Comment:
**Suggestion - Non-blocking**: Could we move this to a different file? just to avoid increasing the size of `wrapper.ts`

Reply:
Sure!